js/content/store.js

```diff
@@ -455,6 +455,25 @@ class StorePageClass {
             })
         });
     }
+
+    forceVideoMP4() {
+        if (!SyncedStorage.get("mp4video")) { return; }
+
+        Array.from(document.querySelectorAll("[data-webm-source]")).forEach(function(node) {
+            let mp4 = node.dataset.mp4Source;
+            let mp4hd = node.dataset.mp4HdSource;
+            if (!mp4 || !mp4hd) return;
+
+            node.dataset.webmSource = mp4;
+            node.dataset.webmHdSource = mp4hd;
+
+            let video = node.querySelector("video");
+            if (!video) { return; }
+
+            video.dataset.sdSrc = mp4;
```

Comment:
The video doesn't have this attribute, thus it won't work for SD playbacks

Reply:
What do you mean? I'm assigning this attribute, for the hd player.

Reply:
I meant that only the parent of the `<video>` uses an attribute named `sdSrc`, the video itself doesn't (look at the page without AS loaded)